src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -3995,18 +3989,12 @@ public bool ReceiveAsync(SocketAsyncEventArgs e)
                 throw new ArgumentNullException(nameof(e));
             }
 
-            // Prepare for the native call.
-            e.StartOperationCommon(this);
-            e.StartOperationReceive();
-
-            // Local vars for sync completion of native call.
-            SocketFlags flags;
+            // Prepare for and make the native call.
+            e.StartOperationCommon(this, SocketAsyncOperation.Receive);
             SocketError socketError;
-
-            // Wrap native methods with try/catch so event args object can be cleaned up.
             try
             {
-                socketError = e.DoOperationReceive(_handle, out flags);
+                socketError = e.DoOperationReceive(_handle, out _);
```

Comment:
I don't think this out param is actually used anywhere.  Can we remove it entirely?

Reply:
Certainly looks that way.  Will remove.